commcare_connect/templates/opportunity/user_learn_progress.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+  <div class="container">
```

Comment:
I'm not quite sure where this info belongs but fine to leave it on a separate page for now.